app/components/UI/FiatOnRampAggregator/Views/Region.js

```diff
@@ -40,7 +40,8 @@ const Region = () => {
 	const [{ data, isFetching, error }] = useSDKMethod('getCountries');
 	const [selectedRegionObject, setSelectedRegionObject] = useState({});
 	const navigation = useNavigation();
-	const handleCheckBox = () => {
+
+	const handleChangeRememberRegion = () => {
 		setRememberRegion(!rememberRegion);
```

Comment:
Use arrow function setState to flip booleans.
```suggestion
		setRememberRegion(currentRememberRegion => !currentRememberRegion);
```

Reply:
That was an eagle-eye comment :) love it. changed